business_objects/record_label_association.py

```diff
@@ -812,6 +812,23 @@ def is_any_record_manually_labeled(project_id: str):
     return value is not None
 
 
+def is_any_record_manually_labeled_by_lt_id(
+    project_id: str, labeling_task_id: str
+) -> bool:
+    query = f"""
+    SELECT rla.id
+    FROM record_label_association rla
+    INNER JOIN labeling_task_label ltl
+        ON rla.labeling_task_label_id = ltl.id AND ltl.project_id = rla.project_id
+    WHERE rla.project_id = '{project_id}'
+    AND ltl.labeling_task_id = '{labeling_task_id}'
+    AND rla.source_type = '{enums.LabelSource.MANUAL.value}'
+    LIMIT 1
+    """
+    value = general.execute_first(query)
+    return value is not None
+
+
```

Comment:
maybe combine with previous and optional parameter.

Reply:
combined